src/c++/perf_analyzer/test_load_manager.cc

```diff
@@ -275,17 +278,17 @@ class TestLoadManager : public TestLoadManagerBase, public LoadManager {
   {
     using time_point = std::chrono::time_point<std::chrono::system_clock>;
     using ns = std::chrono::nanoseconds;
-    auto timestamp1 =
-        std::make_tuple(time_point(ns(1)), time_point(ns(2)), 0, false);
-    auto timestamp2 =
-        std::make_tuple(time_point(ns(3)), time_point(ns(4)), 0, false);
-    auto timestamp3 =
-        std::make_tuple(time_point(ns(5)), time_point(ns(6)), 0, false);
-
-    SUBCASE("No threads")
-    {
-      CHECK(CountCollectedRequests() == 0);
-    }
+    auto timestamp1 = std::make_tuple(
+        time_point(ns(1)), std::vector<time_point>{time_point(ns(2))}, 0,
+        false);
```

Comment:
still need to update my system to enable the new formatting standard. I will try to fix this but it should be good content wise.